tests/python/rest_api/test_tasks.py

```diff
@@ -6349,3 +6349,72 @@ def check_element_outside_count(track_idx, element_idx, expected_count):
         check_element_outside_count(1, 0, 1)
         check_element_outside_count(1, 1, 2)
         check_element_outside_count(1, 2, 2)
+
+
+@pytest.mark.usefixtures("restore_db_per_class")
+@pytest.mark.usefixtures("restore_redis_ondisk_per_function")
+@pytest.mark.usefixtures("restore_redis_ondisk_after_class")
+@pytest.mark.usefixtures("restore_redis_inmem_per_function")
+class TestPatchExportFrames(TestTaskData):
+
+    @fixture(scope="class")
+    @parametrize("media_type", [_SourceDataType.images, _SourceDataType.video])
+    @parametrize("step", [5])
+    @parametrize("frame_count", [20])
+    @parametrize("start_frame", [None, 3])
+    def fxt_uploaded_media_task(
+        self,
+        request: pytest.FixtureRequest,
+        media_type: _SourceDataType,
+        step: int,
+        frame_count: int,
+        start_frame: Optional[int],
+    ) -> Generator[tuple[_TaskSpec, Task, str], None, None]:
+        args = dict(request=request, frame_count=frame_count, step=step, start_frame=start_frame)
+
+        if media_type == _SourceDataType.images:
+            (spec, task_id) = next(self._uploaded_images_task_fxt_base(**args))
+        else:
+            (spec, task_id) = next(self._uploaded_video_task_fxt_base(**args))
+
+        with make_sdk_client(self._USERNAME) as client:
+            task = client.tasks.retrieve(task_id)
+
+        yield (spec, task, f"CVAT for {media_type} 1.1")
```

Comment:
```suggestion
            yield (spec, task, f"CVAT for {media_type} 1.1")
```

Later there is a call to `get_meta()`, it uses the client.

Reply:
You mean, it's better to have it inside the fixture as well?

Reply:
The client is created inside the fixture, but it's context manager is closed before the fixture yields the value. If you want to use the client (saved in the `task` object), don't close the context manager before `yield`.